crypto/dkg_test.go

```diff
@@ -1,3 +1,4 @@
+//go:build relic
 // +build relic
```

Comment:
Do we need both of these?

Reply:
IDE has been adding them automatically lately, though the `// +` seems enough.